logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -617,9 +617,12 @@ class ConversationDataSource(
      * Fetches a list of all recipients for a given conversation including this very client
      */
     override suspend fun getConversationRecipients(conversationId: ConversationId): Either<CoreFailure, List<Recipient>> =
-        getConversationMembers(conversationId).map { it.map(idMapper::toApiModel) }.flatMap {
-            wrapApiRequest { clientApi.listClientsOfUsers(it) }.map { memberMapper.fromMapOfClientsResponseToRecipients(it) }
-        }
+        getConversationMembers(conversationId)
+            .map { it.map(idMapper::toDaoModel) }
+            .flatMap {
+                wrapStorageRequest { clientDAO.getClientsOfUsersByQualifiedIDs(it) }
+                    .map { memberMapper.fromMapOfClientsEntityToRecipients(it) }
+            }
```

Comment:
We can move this logic one layer down to the DB, something like inside Conversations.sq

```
SELECT user_id, GROUP_CONCAT(id) FROM Client WHERE id IN (SELECT user FROM Member WHERE conversation = ?) GROUP BY user_id;
```
This way we have one db read operation to get Recipients